src/components/HeroSection/HeroSection.css

```diff
@@ -0,0 +1,91 @@
+.hero-section {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  gap: 100px;
+}
+.hero-section .hero-content {
+  display: flex;
+  flex-direction: column;
+  row-gap: 50px;
+}
+.hero-section .hero-content .hero-title .hero-title-content {
+  font-family: "Playfair Display", sans-serif;
+  font-weight: 700;
+  font-size: 64px;
+  line-height: 80px;
+  color: #202336;
+}
+.hero-section .hero-content .hero-form-title .hero-form-title-content {
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-size: 22px;
+  color: #202336;
+  line-height: 80px;
+}
+.hero-section .hero-content .rectangle-bar {
+  width: 30px;
+  height: 2px;
+  background: #202336;
+  border-radius: 4px;
+}
+.hero-section .hero-content .hero-form {
+  display: grid;
+  grid-template: 1fr 1fr/1fr 1fr;
+  -moz-column-gap: 30px;
+       column-gap: 30px;
+  row-gap: 35px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+}
+.hero-section .hero-content .hero-form .hero-form-selection {
+  -webkit-appearance: none;
+     -moz-appearance: none;
+          appearance: none;
+  background: url("images/dropdown.png") no-repeat;
+  background-position: right 30px top 23px;
+  line-height: 32px;
+  color: #4A4C53;
+  width: 220px;
+  height: 50px;
+  border: 0.8px solid #AFB0B9;
```

Comment:
- [ ] For good practice, when using pixel values always make sure it's a whole number.

Reply:
Fixed